intercom-java/src/main/java/io/intercom/api/Company.java

```diff
@@ -258,15 +300,14 @@ public int getSessionCount() {
     }
 
     /**
-     * Deprecated. The Intercom API does not support changing the
-     * session value for a company. Calling this method has no
-     * effect.
+     * Note. The Intercom API does not support changing the
+     * session value for a company. Method needed for testing
      *
-     * @param sessionCount this value is ignored
+     * @param sessionCount
      * @return the company object
      */
-    @Deprecated
```

Comment:
Needed to remove this as it was preventing tests from passing. 
Open to alternative approaches to this

It is possible to remove `setSessionCount method` completely and Jackson will automatically read and extract the correct `sessionCount` value